### ISSUE TITLE
Compatibility issues with legacy systems

### ISSUE THREAD

**Issue by davidep** [EricZimmerman/AppCompatCacheParser#3]
I'm trying to run it remotely on a system with Windows Server 2008 R2 Standard Service Pack 1, 64-bit and it doesn't run.

I have no issues with other OS that I'm testing with same method.

Unfortunately I have no GUI or additional logs that can help, but I guess some dll maybe missing on this OS version.

The issue persist with the latest version 1.4.2 but also with older version such as 0.9.8

**Comment by EricZimmerman** [comment]
You need .net 4.6 for it to work. I don't think that's supported on that.
Afaik .net 4.52 is the latest for 2008. Use kape to extract the system hive
and go from there perhaps?

**Comment by davidep** [comment]
Yep I have other ways to extract the hive and parse it locally, I just wanted to make sure that processing locally was not an option.
Many thanks for the quick answer Eric.

**Comment by EricZimmerman** [comment]
I may be able to compile the tool using .net 452 which would fix your
situation too but not until next week I'm guessing.

**Comment by davidep** [comment]
don't worry, would be a nice to have also for the future but not a blocker. Thanks again ;)

**Comment by EricZimmerman** [comment]
so i took a look at this, but the Registry project needs the newer framework as of right now. ill keep digging and if i can back it a bit more backward compatible i will do so and then update this to run on 4.5.2
